sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -581,62 +586,6 @@ private[sql] object ParquetFileFormat extends Logging {
     }
   }
 
-  /** This closure sets various Parquet configurations at both driver side and executor side. */
-  private[parquet] def initializeLocalJobFunc(
```

Comment:
@davies Do you mind if I ask a question please? So, does removing overrideMinSplitSize mean  https://issues.apache.org/jira/browse/SPARK-10143 is not happening anymore? I thought this is a regression that we don't use this function.

I am pretty sure that you took a look for this as well but just want to double check and for my curiosity.